scene/gui/spin_box.cpp

```diff
@@ -244,6 +254,20 @@ String SpinBox::get_prefix() const {
 	return prefix;
 }
 
+void SpinBox::set_update_on_text_changed(bool p_update) {
```

Comment:
Should return when there's nothing to do, otherwise you'll get errors when connecting and already connected signal if you call the method twice.